app/api/schema/tickets.py

```diff
@@ -57,6 +57,11 @@ def validate_quantity(self, data):
                 raise UnprocessableEntity({'pointer': '/data/attributes/quantity'},
                                           "quantity should be greater than or equal to min-order")
 
+        if 'min_price' in data and 'max_price' in data:
+                if data['min_price'] > data['max_price']:
+                    raise UnprocessableEntity({'pointer': '/data/attributes/quantity'},
```

Comment:
This exception logic should only apply for the donation tickets, type check for them too.